policyengine_us/variables/gov/states/ca/chhs/is_ca_state_medicaid_adult_eligible.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class is_ca_state_medicaid_adult_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "California state-funded Medicaid adult eligible"
+    definition_period = YEAR
```

Comment:
```suggestion
    definition_period = YEAR
    defined_for = StateCode.CA
```

Reply:
I think we should limit all of the CA specific vars to state code CA limit computation trees